components/admin_console/feature_discovery/feature_discovery.tsx

```diff
@@ -78,29 +79,6 @@ export default class FeatureDiscovery extends React.PureComponent<Props, State>
         this.props.actions.getPrevTrialLicense();
     }
 
-    requestLicense = async (e: React.MouseEvent) => {
-        e.preventDefault();
-        if (this.state.gettingTrial) {
-            return;
-        }
-        this.setState({gettingTrial: true, gettingTrialError: null});
-        let users = 0;
-        if (this.props.stats && (typeof this.props.stats.TOTAL_USERS === 'number')) {
-            users = this.props.stats.TOTAL_USERS;
-        }
-        const requestedUsers = Math.max(users, 30);
-        const {error, data} = await this.props.actions.requestTrialLicense(requestedUsers, true, true, this.props.featureName);
```

Comment:
Can we remove `actions.requestTrialLicense` and `actions.getLicenseConfig` from the props and from the index file that passes it in?